doc/guides/glossary.rst

```diff
@@ -0,0 +1,117 @@
+
+========
+Glossary
+========
+
+.. _glossary_image_pixel:
+
+:dcm:`Image Pixel Module<part03/sect_C.7.6.3.html>`
+===================================================
+
+**(0028,0002) Samples per Pixel**
+    The number of samples per pixel, otherwise known as the number of image
+    channels, components or planes. An RGB image has 3 samples per pixel (red,
+    green and blue), a grayscale image has 1 sample per pixel (intensity).
+    The *Samples per Pixel* for all DICOM *Pixel Data* is either 1 or 3,
+    however 4 was previously allowed.
+
+    Allowed values: ``1`` or ``3``, but may be constrained by the :dcm:`IOD
+    <part03/ps3.3.html>`.
+
+**(0028,0004) Photometric Interpretation**
+    The intended interpretation of the *Pixel Data* in its *current form* in
+    the dataset. For example:
+
+    * If you have a dataset with RGB *Pixel Data* then the  *Photometric
+      Interpretation* should be ``'RGB'``.
+    * If you take your RGB data and convert it to `YCbCr
+      <https://en.wikipedia.org/wiki/YCbCr>`_ then the *Photometric
+      Interpretation* should be ``'YBR_FULL'`` (or a related interpretation
+      depending on the conversion method).
+    * If you then compress that data using *RLE Lossless* encoding then the
+      *Photometric Interpretation* remains ``'YBR_FULL'``.
+    * On the other hand, if you take your original RGB data and apply *JPEG
+      2000 Lossless* encoding then the *Photometric Interpretation* will either
+      be ``'RGB'`` or ``'YBR_RCT'`` depending on whether or not the encoder
+      performs multiple component transformation.
+
+    When compressing pixel data using one of the JPEG encodings it's important
+    to know if the encoder is performing any color space transformation prior
+    to compression, as this needs to be taken into account when setting
+    the *Photometric Interpretation*. This reason this is important is that
```

Comment:
This reason -> The reason why

Reply:
Maybe split this long sentence up for better readability (it sounds like a typical German sentence 😁)

Reply:
I cut the length down instead. Had a hard time splitting it while keeping it coherent.

Reply:
Thanks, I like this better - and yes, I didn't provide a split version for a reason 😁